modules/test/playwright/tests/frontend-data-set-views-web/pages/view/tabs/ActionsPage.ts

```diff
@@ -137,9 +135,9 @@ export class ActionsPage {
 				? `${actionProps.name} title`
 				: `${actionProps.title}`;
 
-			await this.page.getByPlaceholder('add-here-the-title').click();
+			await this.page.getByPlaceholder('Add the title').click();
```

Comment:
Uff, we should have fixed this when we added the test :disappointed: 

Nice cleanup!